tests/pass-dep/libc/libc-epoll-no-blocking.rs

```diff
@@ -254,6 +258,34 @@ fn test_epoll_eventfd() {
     let expected_event = u32::try_from(libc::EPOLLIN | libc::EPOLLOUT).unwrap();
     let expected_value = u64::try_from(fd).unwrap();
     check_epoll_wait::<8>(epfd, &[(expected_event, expected_value)]);
+
+    // Write to the eventfd again.
+    let res = unsafe { libc_utils::write_all(fd, sized_8_data.as_ptr() as *const libc::c_void, 8) };
+    assert_eq!(res, 8);
+
+    // This does not change the status, so we should get no event.
+    // However, Linux performs a spurious wakeup.
+    check_epoll_wait::<8>(epfd, &[(expected_event, expected_value)]);
```

Comment:
... This (setting `force_edge` to `false`) makes it so that we get the empty event list here. Which is what I would expect since the readiness did not change. However, Linux does seem to always emit an epoll event on eventfd writes, even if readiness does not change -- and it looks like Tokio relies on that? At least that is the only explanation I have for the deadocks I have been seeing.

Reply:
@Darksonn FWIW it seems like Linux triggers a wakeup even if you write `[0u8; 8]` to the eventfd. That way you'd never even have to worry about overflow.